peps/pep-0750.rst

```diff
@@ -0,0 +1,813 @@
+PEP: 750
+Title: Tag Strings For Writing Domain-Specific Languages
+Author: Jim Baker <jim.baker@python.org>, Guido van Rossum <guido@python.org>, Paul Everitt <pauleveritt@me.com>
+Sponsor: Lysandros Nikolaou <lisandrosnik@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 01-Jun-2024
+Python-Version: 3.14
+
+Abstract
+========
+
+This PEP introduces tag strings for custom, repeatable string processing. Tag strings
+are an extension to f-strings, with a custom function -- the "tag" -- in place of the
+``f`` prefix. This function can then provide rich features such as safety checks, lazy
+evaluation, domain-specific languages (DSLs) for web templating, and more.
+
+Tag strings are similar to `JavaScript tagged template literals <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates>`_
+and related ideas in other languages. The following tag string usage shows how similar it is to an ``f`` string, albeit
+with the ability to process the literal string and embedded values:
+
+.. code-block:: python
+
+    name = "World"
+    greeting = greet"hello {name}"
+    assert greeting == "Hello WORLD!"
+
+
+Tag functions accept prepared arguments and return a string:
+
+.. code-block:: python
+
+    def greet(*args):
+        salutation, recipient, *_ = args
+        _, getvalue = recipient
```

Comment:
And elsewhere? 
```suggestion
        _, get_value = recipient
```

Reply:
This spelling came from a preference expressed when Jim/Guide/I chatted two PyCon's ago. I suggest we leave it as-is, but if others prefer `get_value`, certainly change it.